src/carousel.tsx

```diff
@@ -138,6 +146,7 @@ const useStyles = createUseStyles((theme) => ({
         },
     },
     item: {
+        scrollSnapStop: isAndroid(theme.platformOverrides) ? 'always' : 'normal',
```

Comment:
This is a small fix for android devices, where the scroll inertia is too high and it's difficult to scroll only one slide without forcing scroll snap to stop.

The default behavior works fine in desktop and in iOS so I've decided to only apply this to Android devices.